### DISCUSSION TITLE
what's the diff with bottles, playonlinux, etc?

### DISCUSSION THREAD

**Discussion by BarbzYHOOL** [Vysp3r/ProtonPlus#57] [Q&A] [answered]
what's the diff with bottles, playonlinux, etc?

**Comment by Vysp3r** [comment]
Could you be a bit more clear?
I can tell you the difference between Bottles and PlayOnLinux, but I can't for all.
Could you maybe rephrase your question?

**Reply by BarbzYHOOL** [reply]
What is the difference between ProtonsPlus and Bottles/PlayonLinux/Lutris?

**Comment by Vysp3r** [comment, marked as answer]
ProtonPlus only purpose is  to manage Wine and Proton compatibility tools for application like Bottles and Lutris. It's not to download and manage games or applications like Bottles and Lutris.

**Reply by BarbzYHOOL** [reply]
ok not sure what it's for but thanks

**Reply by Vysp3r** [reply]
What part don't you understand? I will try my best to explain it better.

**Reply by BarbzYHOOL** [reply]
it's for installing proton versions in bottles, lutris etc?? i have not yet used proton, but again we need to have different versions of proton?? the madness never ends with wine lol

**Reply by Vysp3r** [reply]
Yeah, some games requires specific fixes that may or may not be included in the version you're currently. To make things easier, people just created alternative versions of Proton that include some fixes. For example, you have Proton-GE which is the most popular one. Most of the time you should be good with the one provided by default with Bottles, Lutris or any other application like that. We're not 100% there on Linux for gaming, but we're getting close. This tool is just to make it easier for people to install and manage their tools in one place.

**Reply by Vysp3r** [reply]
I guess you're not using Linux as your main OS if you still haven't used proton?

**Reply by BarbzYHOOL** [reply]
na in fact, i don't play games but sometimes I want to play one or install one for a relative and going back to wine endless issues

thx for answers

**Reply by ptkato** [reply]
How would it work in Lutris, for example, since it has its own wine/proton manager and also scripts that download the wine/proton automatically?

I mean, not how it would work, but the intent really, why would I want to use this tool rather than what is already embedded within Lutris?

**Reply by Vysp3r** [reply]
It's mostly out of ease of use and how convenient it is. Having a place where I can manage all of the launchers in one place is nice and what if you want to install a wine version that's not in the current manager from Lutris. I plan on adding a lot more features that will make the app even more useful in the future like the notifications tab where you could receive notifications about when a new version from a tool is available. The games tab will display all your games from each launcher, which will show lots of useful data and probably more cool stuff for each games.
